phalcon/mvc/model.zep

```diff
@@ -3853,7 +3853,7 @@ abstract class Model implements EntityInterface, ModelInterface, ResultInterface
 	 *
 	 * @param string|array fieldName
 	 */
-	public function hasChanged(var fieldName = null) -> boolean
+	public function hasChanged(var fieldName = null, boolean allFields = false) -> boolean
```

Comment:
Yea i will update changelog once there will be 3.2.x branch cuz i will need to update it then anyway.